repo-projects/deployment-tools.proj

```diff
@@ -1,5 +1,4 @@
 <Project Sdk="Microsoft.Build.NoTargets">
-
```

Comment:
Please let's not work against each other. I like to add these new lines after project opening tags and before project closing tags (and our dotnet new templates have them as well) and I noticed you tend to remove them. I really think that the new lines here improves readability of msbuild files.

just a nit observation